src/ucp/core/ucp_mm.c

```diff
@@ -296,6 +368,19 @@ ucs_status_t ucp_mem_unmap(ucp_context_h context, ucp_mem_h memh)
     return status;
 }
 
+ucs_status_t ucp_mem_query(const ucp_mem_h memh, ucp_mem_attr_t *attr)
+{
+    if (memh->address) {
+        attr->field_mask = UCP_MEM_ATTR_FIELD_ADDRESS |
```

Comment:
yes, see ucp_worker_query for example.
if not available - currently it's not handled correctly (good catch)
I guess we need to return ERR_INVALID_PARAM if any unexpected bit is received in attr.field_mask. but let's do it in another PR, and for all ucp_*_query() functions?